domain/src/main/java/com/teamagam/gimelgimel/domain/map/entities/symbols/BaseSymbol.java

```diff
@@ -2,14 +2,24 @@
 
 public abstract class BaseSymbol implements Symbol {
 
-  private boolean mIsSelected;
+  private final boolean mIsSelected;
+  private final String mText;
 
-  public BaseSymbol(boolean isSelected) {
+  BaseSymbol(boolean isSelected, String text) {
```

Comment:
You can add constructor without the text parameter, so you won't pass null every time.